{{cookiecutter.project_slug}}/package.json

```diff
@@ -1,26 +1,46 @@
 {
   "private": true,
-  "dependencies": {
-    "autoprefixer": "9.3.1",
+  "browserslist": [
+    "> 1%",
+    "last 2 versions"
+  ],
+  "devDependencies": {
+    "autoprefixer": "7.1.6",
+    "babel-core": "6.26.0",
+    "babel-loader": "7.1.2",
+    "babel-preset-env": "1.7.0",
     "browser-sync": "2.26.3",
-    "cssnano": "4.1.7",
-    "gulp": "3.9.1",
-    "gulp-postcss": "8.0.0",
-    "gulp-sass": "4.0.2",
-    "gulp-sourcemaps": "2.6.4",
-    "jshint": "2.9.6",
-    "normalize-scss": "7.0.1",
-    "postcss-import": "12.0.1",
+    "browser-sync-webpack-plugin": "2.2.2",
+    "css-loader": "0.28.7",
+    "cssnano": "^4.1.8",
+    "es6-promise": "4.1.1",
+    "eslint": "5.7.0",
+    "eslint-config-airbnb-base": "13.1.0",
+    "eslint-config-prettier": "^3.3.0",
+    "eslint-loader": "2.1.0",
+    "eslint-plugin-import": "2.14.0",
+    "eslint-plugin-prettier": "^3.0.0",
+    "node-sass": "4.5.3",
+    "normalize-scss": "7.0.0",
+    "postcss-loader": "2.0.8",
     "prettier": "1.15.2",
-    "pump": "3.0.0",
-    "yargs-parser": "11.1.1"
+    "sass-loader": "6.0.6",
+    "style-loader": "0.19.0",
+    "stylelint": "8.2.0",
+    "stylelint-config-standard": "17.0.0",
+    "stylelint-selector-bem-pattern": "2.0.0",
+    "stylelint-webpack-plugin": "0.9.0",
+    "url-loader": "^1.1.2",
+    "vue": "^2.5.20",
```

Comment:
Remove `^` from all of these - npm tends to rewrite the lock file at times because of it.